File: 3227-find-missing-and-repeated-values/3227-find-missing-and-repeated-values.py
class Solution:
    def findMissingAndRepeatedValues(self, arr: List[List[int]]) -> List[int]:
        st = set()

        n = len(arr)
        dup, miss = -1,-1
        for i in range(n):
            for j in range(n):
                if arr[i][j] in st:
                    dup = arr[i][j]
                st.add(arr[i][j])

        print(st)
        for i in range(1,pow(n,2)+1):
            print(i)
            if i not in st:
                miss = i
                break

        return [dup,miss]

File: 3227-find-missing-and-repeated-values/3227-find-missing-and-repeated-values.cpp
class Solution {
public:
    vector<int> findMissingAndRepeatedValues(vector<vector<int>>& arr) {
        unordered_set<int> st;
        int dup = -1, miss= -1;
        for(int i=0; i < arr.size(); i++){
            for(int j =0; j < arr.size(); j++){
                if(st.find(arr[i][j])!=st.end()){
                    dup = arr[i][j];
                    continue;
                }
                st.insert(arr[i][j]);
            }
        }
        for(int i = 1; i<=pow(arr.size(),2); i++){
            if(miss == -1 && st.find(i)==st.end()){
                miss = i;
                break;
            }  
        }
        return {dup,miss};
    }
};